openlibrary/catalog/add_book/merge.py

```diff
@@ -12,13 +12,6 @@ def db_name(a):
         date = a.date
     return ' '.join([a['name'], date]) if date else a['name']
 
-def undelete_author(a):
```

Comment:
Are we sure this code also appears to be in `catalog/importer/import_marc.py`, `catalog/importer/update.py` and `catalog/utils/edit.py` fwiw